classes/Rijndael.php

```diff
@@ -48,24 +48,41 @@ public function encrypt($plaintext)
         if ($length >= 1048576) {
             return false;
         }
-        return base64_encode(mcrypt_encrypt(MCRYPT_RIJNDAEL_128, $this->_key, $plaintext, MCRYPT_MODE_CBC, $this->_iv)).sprintf('%06d', $length);
+        $ciphertext = null;
+        if (function_exists('mcrypt_encrypt') && version_compare(phpversion(), '7.1', '<')) {
+            $ciphertext = mcrypt_encrypt(MCRYPT_RIJNDAEL_128, $this->_key, $plaintext, MCRYPT_MODE_CBC, $this->_iv);
+        } elseif (function_exists('openssl_encrypt') && version_compare(phpversion(), '7.1', '>=')) {
```

Comment:
why not testing first openssl_encrypt? you'd not need the version_compare part anymore
e.g:
if (function_exists('openssl_encrypt')) {

} elseif (function_exists('mcrypt_encrypt')) {

} else {
  throw new runtimeException(...);
}